package.json

```diff
@@ -24,12 +24,15 @@
     "stack-trace": "0.0.9",
     "underscore": "1.7.0",
     "unifiedpush-node-sender": "0.12.0",
-    "winston": "0.8.0"
+    "winston": "0.8.0",
+    "xtend": "4.0.1"
```

Comment:
Allows us to inherit (xtend) object properties in a "pure" manner. lodash functions modify the underlying objects which might be an undesired side effect.